src/Package/MSTest.nuspec

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>MSTest</id>
+    <version>1.1.17</version>
```

Comment:
I see it is defined as the other files but shouldn't this be `2.2.10`? If the version is always replaced during build, I would vote for (ok to be a follow up PR) making it more explicit: blank, `42.42.42`, `version-set-by-build`...

Reply:
Version is always replaced during the build. I agree with you, but let’s do that in a separate Pr?